console_input.py

```diff
@@ -0,0 +1,21 @@
+"""
+Purpose: Create a class that ask for string and return it.
+Auther: Roza Hadid
+Date: 17.3.2024
+"""
+from Iinput import InputClass
+
+
+class ConsoleInput(InputClass):
+    """
+    Class that heir from InputClass.
+    Function that return a string that takes from console input.
+    """
+
+    def read(self) -> str:
+        """
+        Function that gets a variable of string type
```

Comment:
Change the documentation, we already know it's a function no need to put it in the documentation